BraveWallet/Crypto/Accounts/Add/AddAccountView.swift

```diff
@@ -24,7 +24,6 @@ struct AddAccountView: View {
       // Add normal account
       let accountName = name.isEmpty ? String.localizedStringWithFormat(Strings.Wallet.defaultAccountName, keyringStore.keyring.accountInfos.filter(\.isPrimary).count + 1) : name
       keyringStore.addPrimaryAccount(accountName) { success in
-        // TODO: Error state
```

Comment:
This won't ever error as long as there's a keyring, which is impossible here (only way to get to this screen is with a keyring created)